__tests__/app.test.js

```diff
@@ -100,3 +100,38 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe.only("GET /api/articles/:article_id/comments", () => {
+  test("should respond with status code 200 and an array of comments if there are any", () => {
```

Comment:
The task says the following:
Comments should be served with the most recent comments first.

You're not testing for that yet! But well done on checking the length